File: Programmers/src/Javascript/연속된수의합.js
function solution(num, total) {
    let min = Math.ceil(total/num - Math.floor(num/2));
    let max = Math.floor(total/num + Math.floor(num/2));
    
    return new Array(num).fill(0).map((a,b) => {
        return b + min;
    });
}

File: Programmers/src/Java/Lv0/연속된 수의 합.java
class Solution {
    public int[] solution(int num, int total) {
        int n = total/num - (num -1)/2;
        int[] answer = new int[num];
        
        for(int i=0; i<num; i++)
            answer[i] = n + i;
        
        return answer;
    }
}